src/handlers/notification/index.js

```diff
@@ -411,7 +411,30 @@ const processMessage = async (msg, span) => {
       histTimerEnd({ success: true, action })
       return response
     } else {
-      Logger.isDebugEnabled && Logger.debug(`Notification::processMessage - Action: ${actionLower} - Skipping notification callback to original sender (${from}) because feature is disabled in config.`)
+      Logger.isDebugEnabled && Logger.debug(`Notification::processMessage - Action: ${actionLower} - Skipping notification callback to original sender (${from}).`)
+    }
+    histTimerEnd({ success: true, action })
+    return true
+  }
+
+  if (actionLower === ENUM.Events.Event.Action.ABORT_VALIDATION) {
```

Comment:
There's no reason to not write a unit test for this at least.

See coverage report: https://9631-129916230-gh.circle-artifacts.com/0/coverage/lcov-report/handlers/notification/index.js.html